Rakefile

```diff
@@ -11,3 +12,12 @@ task :wait_for_router do
 
   abort "Router has no routes after 60 seconds" unless outcome
 end
+
+task :setup_rabbitmq_rummager do
```

Comment:
Do you think this is something that we could set up inside the projects themselves? It feels like it would be beneficial in other envs (pretty useful on vm)

Reply:
It would be good to leave this here so we aren't blocked by deploying out rummager/publishing-api and extract this out as separate PRs.